.htmltest.yml

```diff
@@ -3,16 +3,28 @@ ExternalTimeout: 30
 IgnoreDirs:
   - ^docs/stable
   - ^docs/dev
+  - ^docs/v1.0
+  - ^docs/v1.1
   - ^docs/v1.2
+  - ^docs/v2.0
+  - ^docs/v2.1
+  - ^docs/v19.1
+  - ^docs/v19.2
+  - ^docs/v20.1
+  - ^docs/releases
```

Comment:
I'm gonna take a wild guess and say the build errors are coming from this line here. Can you try omitting to see if that fixes it?

Reply:
Trying this right now.

Reply:
Looks like that fixed it. I think the problem is if we add a directory to that list then any file not being ignored that points to a file that is ignored throws an error, I think? Seems buggy.